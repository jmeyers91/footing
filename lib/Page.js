"use strict";
var __makeTemplateObject = (this && this.__makeTemplateObject) || function (cooked, raw) {
    if (Object.defineProperty) { Object.defineProperty(cooked, "raw", { value: raw }); } else { cooked.raw = raw; }
    return cooked;
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var styled_components_1 = __importDefault(require("styled-components"));
var Column_1 = __importDefault(require("./Column"));
var Page = styled_components_1.default(Column_1.default)(templateObject_1 || (templateObject_1 = __makeTemplateObject(["\n  height: 100vh;\n  width: 100vw;\n  overflow-x: hidden;\n  overflow-y: auto;\n"], ["\n  height: 100vh;\n  width: 100vw;\n  overflow-x: hidden;\n  overflow-y: auto;\n"])));
exports.default = Page;
var templateObject_1;
//# sourceMappingURL=Page.js.map