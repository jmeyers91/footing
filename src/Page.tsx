import styled from 'styled-components';
import Column from './Column';

const Page = styled(Column)`
  height: 100vh;
  width: 100vw;
  overflow-x: hidden;
  overflow-y: auto;
`;

export default Page;
