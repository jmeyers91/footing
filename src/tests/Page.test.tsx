import React from 'react';
import { render } from '@testing-library/react';
import Page from '../../src/Page';

describe('Page', () => {
  test('Should match snapshot', () => {
    const { container } = render(<Page />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
