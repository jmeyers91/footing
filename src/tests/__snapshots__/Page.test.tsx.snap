// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Should match snapshot 1`] = `
<div
  class="sc-bdVaJa sc-bwzfXH glpLfC"
/>
`;
